starter-code/scripts/article.js

```diff
@@ -25,6 +36,10 @@ Article.prototype.toHtml = function() {
     3. article title,
     4. article body, and
     5. publication date. */
+  
+
+  var $author = $newArticle.find(rawData.title).append();
```

Comment:
I don't think you need this because you are getting/setting the article title on line 27.